code/modules/client/client_procs.dm

```diff
@@ -533,4 +534,49 @@ client/verb/character_setup()
 		winset(src, "mainwindow", "menu=menu;statusbar=true")
 		winset(src, "mainwindow.mainvsplit", "pos=3x0")
 	winset(src, "mainwindow", "is-maximized=true")
-	OnResize()
```

Comment:
Why remove OnResize()?

Reply:
I don't really see a reason to have it here :?